src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -264,30 +265,22 @@ public void testUpdateFeedbackResponseComment() throws Exception {
     }
 
     @Test
-    public void testDeleteFeedbackResponseComment() throws Exception {
+    public void testDeleteFeedbackResponseCommentById() throws Exception {
         //create a frComment to delete
         FeedbackResponseCommentAttributes frComment = restoreFrCommentFromDataBundle("comment1FromT1C1ToR1Q1S1C1");
-        frComment.setId(null);
-        frComment.feedbackQuestionId = getQuestionIdInDataBundle("qn2InSession1InCourse1");
-        frComment.feedbackResponseId = getResponseIdInDataBundle("response2ForQ2S1C1", "qn2InSession1InCourse1");
 
         frcLogic.createFeedbackResponseComment(frComment);
 
         ______TS("silent fail nothing to delete");
 
-        frComment.feedbackResponseId = "invalid responseId";
-        //without proper frCommentId and its feedbackResponseId,
-        //it cannot be deleted
-        frcLogic.deleteFeedbackResponseComment(frComment);
+        frcLogic.deleteFeedbackResponseCommentById(1234567L);
+
+        ______TS("typical success case");
 
         FeedbackResponseCommentAttributes actualFrComment =
                 frcLogic.getFeedbackResponseCommentForSession(
                                  frComment.courseId, frComment.feedbackSessionName).get(1);
-        verifyPresentInDatastore(actualFrComment);
-
-        ______TS("typical success case");
-
-        frcLogic.deleteFeedbackResponseComment(actualFrComment);
+        frcLogic.deleteFeedbackResponseCommentById(actualFrComment.getId());
         verifyAbsentInDatastore(actualFrComment);
 
         ______TS("typical success case for response");
```

Comment:
This test doesn't test `deleteFeedbackResponseCommentById`

Reply:
yes, moved this to a new test for `deleteFeedbackResponseCommentsForResponse`